crates/bevy_ptr/README.md

```diff
@@ -109,3 +110,8 @@ accessing elements in the slice is `unsafe`. In debug builds, the length is incl
 `OwningPtr<'a>`, `Ptr<'a>`, and `PtrMut<'a>` act like `NonNull<()>`, but attempts to restore much of the safety guarantees of `Unique<T>`, `&T`, and `&mut T`.
 They allow working with heterogenous type erased storage (i.e. ECS tables, typemaps) without the overhead of dynamic dispatch in a manner that progressively
 translates back to safe borrows. These types also support optional alignment requirements at a type level, and will verify it on dereference in debug builds.
+
+`MovingPtr<'a, T>` is like a lifetimed-`Box<T>` or a typed `OwningPtr<'a>` made for cheaply moving potentially large values around in memory.
+It's a pointer that owns the value it points to but does not own the allocation. If dropped, it will drop the value it points to if it's dropped, just as
```

Comment:
Accidentally left in "If dropped" twice in this sentence :) I'd remove the second one.

```suggestion
It's a pointer that owns the value it points to but does not own the allocation. If dropped, it will drop the value it points to, just as
```